lib/base/timer.cpp

```diff
@@ -61,6 +62,15 @@ static int l_AliveTimers = 0;
 
 static Defer l_ShutdownTimersCleanlyOnExit (&Timer::Uninitialize);
 
+Timer::Ptr Timer::Create()
+{
+	Ptr t (new Timer());
```

Comment:
[`std::make_shared`](https://en.cppreference.com/w/cpp/memory/shared_ptr/make_shared) should be preferred here as this allows for a single allocation for both the object and the control block.

Reply:
Too bad Timer#Timer() is private.